packages/react/src/PageHeader/PageHeader.test.tsx

```diff
@@ -199,7 +207,7 @@ describe('Page Header', () => {
 
     render(<PageHeader logoLinkComponent={CustomLink} />)
 
-    const customLink = screen.getByRole('link', { name: 'Ga naar de homepage' })
+    const customLink = screen.getByRole('link', { name: 'Gemeente Amsterdam logo Ga naar de voorpagina' })
```

Comment:
"voorpagina" is geen woord dat we gebruiken toch? homepage is prima.

Reply:
We sluiten hiermee aan op de [Schrijfwijzer](https://www.amsterdam.nl/schrijfwijzer/heldere-taal-basis-onze-huisstijl/taalgebruik-heldere-taal/woordgebruik-jargon-buitenlandse-woorden/):
> Nederlands is de enige voertaal binnen de Stijl van Amsterdam. Een manager is gewoon een leidinggevende, teamleider, afdelingshoofd, directeur enzovoort. Assets zijn bedrijfsmiddelen. Je gebruikt alleen leenwoorden waar wij geen Nederlands woord voor hebben, zoals: computer, iPad, enzovoort.

Reply:
Ok, maar "voorpagina" blijft niet heel gangbaar toch? Misschien voor een papieren krant. Wikipedia heeft nog een aantal andere opties: https://nl.wikipedia.org/wiki/Homepage Mijn keuze zou uitgaan naar hoofdpagina maar soit.